config/crd/bases/activemonitor.keikoproj.io_healthchecks.yaml

```diff
@@ -30,12 +28,10 @@ spec:
   group: activemonitor.keikoproj.io
   names:
     kind: HealthCheck
-    listKind: HealthCheckList
```

Comment:
also why this is removed.. is this an generated code that removed it?

Reply:
reverted these changes.